alpakka/src/test/scala/com/gu/scanamo/ScanamoAlpakkaSpec.scala

```diff
@@ -145,7 +147,7 @@ class ScanamoAlpakkaSpec
       import com.gu.scanamo.syntax._
 
       val forecasts = for {
-        _ <- ScanamoAlpakka.update(alpakkaClient)("forecast")('location -> "London", set('weather -> "Sun"))
+        _ <- ScanamoAlpakka.update[Forecast](alpakkaClient)("forecast")('location -> "London", set('weather -> "Sun"))
```

Comment:
Not adding the type generates a diverging implicit error, but I see this is used in other places so assuming that's not too bad.

Reply:
I'm OK with it in this case. I'd be a lot more worried if it was one of the methods on `Table`. Just need to think about how to make it clear in release notes.